Build-FFmpeg.ps1

```diff
@@ -303,6 +303,25 @@ if (!(Test-Path $BashExe)) {
     }
 }
 
+# Check for YASM and download if needed
+
+New-Item -ItemType Directory -Force $PSScriptRoot\Output\yasm -OutVariable yasm
+
+if (!(Test-Path $yasm\yasm.exe))
+{
+    if ($Env:PROCESSOR_ARCHITECTURE -eq "AMD64")
+    {
+        Start-BitsTransfer -Source http://www.tortall.net/projects/yasm/releases/yasm-1.3.0-win64.exe -Destination $yasm\yasm.exe

```

Comment:
Why "BitsTransfer" instead of `Invoke-WebRequest`?

Reply:
I read that it's faster. Not needed anyways if I can get yasm through pacman.